_handbook/onboarding.md

```diff
@@ -6,96 +6,106 @@ shorthand: Onboarding
+- GSTT Microsoft Teams i.e., to access secondary CSC chat, GSTT-shareable only files, GSTT calendar, etc.
+- 10X Workshops meeting series (fortnightly in-person sessions 3 - 5:30PM i.e., every other Wednesday)
+- Bi-weekly stand-ups (9:30 - 10AM on Tuesdays and Thursdays)
+- Quality Management System (QMS) audits (internal and external)
+- Interesting project meetings
+- Conferences
 
-There is a central room booking service allowing GSTT staff to book a variety of
-meeting rooms (without charge) across our sites including Minerva and Great Dover
-Street.
+**CSC Desk Area**
 
-Go to [Room Booking](https://gstt.nhs.micadipr.net/pages/roomMonitoringClients/rmSignIN.asp?return=%2Fpages%2FroomMonitoringClients%2FrmRoomSearchForClients%2Easp%3F) from the GSTT network (using Citrix if not onsite)
+The CSC team mainly resides on the 10th Floor in the [Education Centre](https://maps.app.goo.gl/J5BUnrPw1eHsNDdq8) (also referred to as **York Road**), but can also be granted access to:
 
+- **Tabard House** ([near Guy's Hospital](https://maps.app.goo.gl/jDNeBb3Q21Bd7eUL8))
+- The 5th ("Big Ben" side) and 9th floor (meeting rooms) in [**Becket House**](https://maps.app.goo.gl/g3Rc6bPowryfpby16) 
+  - For access, please email the Operations Assistant for the Biomedical Engineering department for the necessary forms, training, KCL card issuance, and guidance.
 
-Simply register an account and complete your request (date/time/room size and function) and a selection of options come 
-up which you can proceed to book online.
+Access to these sites should be requested and approved by your line manager, and implemented so you can simply swipe your card at the entrance. 
 
-**Electronic Staff Record**
+**Access to GSTT-specific websites**
 
-In order to check payslips, p60, pension contributions etc. you need to get access to your Electronic Staff Record.
-   - Browse https://my.esr.nhs.uk/dashboard/web/esrweb
-   - Go to login to ESR
-   - Click Unlock Account if is your first time
-   - Fill in details and wait for an email with instructions. It will usually be your username and a link to set up your password
+New starters should ensure they register and have access to:
 
-**Citrix Access**
+- [MICAD room booking system](https://gstt.nhs.micadipr.net/pages/roomMonitoringClients/rmSignIN.asp?return=%2Fpages%2FroomMonitoringClients%2FrmRoomSearchForClients%2Easp%3F)* i.e., to book meeting rooms across GSTT sites.
+- [Electronic Staff Record (EST)](https://my.esr.nhs.uk/dashboard/web/esrweb) i.e., to access payslips, P60 forms, pension contributions, etc.
+  - Click on "Unlock Account" to register for the first time.
+- [HR Portal](https://fids.gstt.nhs.uk/adfs/ls/)** i.e., to raise and manage HR requests/transactions, access HR policies, etc.
+- [IT Service Centre](https://gstt.service-now.com/ess/?id=my_requests)** i.e., to raise and manage IT requests/incidents, access the IT system catalogue, etc.
```

Comment:
I believe this is linked to the GSTT NHS email account which should be requested by the line manager of the new starter, the same GSTT account is used for the IT Service Centre